### PR TITLE
hw_07

### PR DESCRIPTION
**Pull request by SvetlanaYurkova**

### CODE REVIEW

**Comment by SvetlanaYurkova** [comment]
hw07

File: hw07.task1/Program.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите стихотворение в одну строку разделяя строки точкой с запятой:");
+            string poem = Console.ReadLine();
+            string[] arrayPoem= ArrayStringsPoem(poem);
+            OutputColumnPoem(arrayPoem);
+            string[] arrayPoemChange = ChangeLetterOtoA(arrayPoem);
```
Comment (VadzimPapko) [inline hw07.task1/Program.cs:13]: ChangeLetterOtoA => possible to use the following naming `ChangeLetterO2A`

File: hw07.task1/Program.cs [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        {
+            string[] arrayStringsPoem = strPoem.Split(';');
+            return arrayStringsPoem;
+        }
+        public static string[] ChangeLetterOtoA(string[] arrayStringsPoem)
+        {
+            for (int i = 0; i < arrayStringsPoem.Length; i++)
+            {
+                arrayStringsPoem[i] = arrayStringsPoem[i].Replace('о', 'а');
+                arrayStringsPoem[i] = arrayStringsPoem[i].Replace('О', 'А');
```
Comment (VadzimPapko) [inline hw07.task1/Program.cs:27]: it's a bit weird - better to use one method before - string.ToUpper()

File: hw07.task1/poem.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+﻿Опрятней модного паркета;Блистает речка, льдом одета.;Мальчишек радостный народ;Коньками звучно режет лед;На красных лапках гусь тяжелый,;Задумав плыть по лону вод,;Ступает бережно на лед,;Скользит и падает веселый;Мелькает, вьется первый снег,;Звездами падая на брег.
```
Comment (VadzimPapko) [inline hw07.task1/poem.txt:1]: 👍

File: hw07.task2/Program.cs [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+                {
+                    min = arrayWords[i].Length;
+                    indMinWord = i;
+                }
+            }
+            return indMinWord;
+        }
+        public static int NumberofPunctuationMarks(string strInput)
+        {
+            int count = 0;
```
Comment (VadzimPapko) [inline hw07.task2/Program.cs:98]: Type System.Char has some good methods such as 
image

File: hw07.task2/Program.cs [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+                        temp = sortdesc[i];
+                        sortdesc[i] = sortdesc[j];
+                        sortdesc[j] = temp;
+                    }
+                }
+            }
+            return sortdesc;
+        }
+    }
+}
```
Comment (VadzimPapko) [inline hw07.task2/Program.cs:127]: logic in general is correct

**Review by VadzimPapko — COMMENTED** [review]
Done. Your score is 9

**Pull request merged**